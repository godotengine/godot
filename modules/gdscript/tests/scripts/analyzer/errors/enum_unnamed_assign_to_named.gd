enum { ENUM_VALUE_1, ENUM_VALUE_2 }

enum MyEnum { ENUM_VALUE_1, ENUM_VALUE_2 }

func test():
	var local_var: MyEnum = ENUM_VALUE_1
	print(local_var)
