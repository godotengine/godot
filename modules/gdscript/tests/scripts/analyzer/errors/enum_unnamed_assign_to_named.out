GDTEST_ANALYZER_ERROR
>> ERROR at line 6: Cannot assign a value of type "enum_unnamed_assign_to_named.gd.<anonymous enum>" as "enum_unnamed_assign_to_named.gd.MyEnum".
>> ERROR at line 6: Cannot assign a value of type enum_unnamed_assign_to_named.gd.<anonymous enum> to variable "local_var" with specified type enum_unnamed_assign_to_named.gd.MyEnum.
